test/integration/testMaxLobSize.js

```diff
@@ -0,0 +1,265 @@
+const assert = require('assert');
+const async = require('async');
+const testUtil = require('./testUtil');
+const crypto = require('crypto');
+const snowflake = require('./../../lib/snowflake');
+const connOption = require('./connectionOptions');
+
+function generateRandomString(sizeInBytes) {
+  const bufferSize = Math.ceil(sizeInBytes / 2);
+  const buffer = crypto.randomBytes(bufferSize);
+  return buffer.toString('hex').slice(0, sizeInBytes);
+}
+
+describe.skip('Max LOB test', function () {
+  let connection;
+  const SMALL_SIZE = 16;
+  const ORIGIN_SIZE = 16 * 1024 * 1024; 
+  const MEDIEUM_SIZE = ORIGIN_SIZE * 2;
+  const LARGE_SIZE = MEDIEUM_SIZE * 2;
+  const MAX_LOB_SIZE = LARGE_SIZE * 2;
+
+  const singleTest = 'select randstr(50000000, 124);';
```

Comment:
If test with this case, make the the size configurable and test with different sizes same as other test cases.

Reply:
I created another testing case that tests select randstr query with the different sizes.